github-intro/tictactoe.css

```diff
@@ -4,4 +4,10 @@ table {
 
 td {
     border: 3px solid gray;
+    min-width: 50px;
+    min-height: 50px;
```

Comment:
I think this should be `height` not `min-height`.

Reply:
I fixed this, thanks for the review!